### PR TITLE
create chatroom

### LINKED ISSUE #3: As a frontend developer, I need api endpoint to create chatrooms for user.

### PR DESCRIPTION
**Pull request by vanvo1605**

### CODE REVIEW

File: chatroom/views.py
Comment (kunikao) [inline chatroom/views.py:1]: why change the create to createat

**Review by kunikao — COMMENTED** [review]
test

**Comment by kunikao** [comment]
test1

File: Main/urls.py
Comment (kunikao) [inline Main/urls.py:1]: why?

**Review by kunikao — APPROVED** [review]
ttt

File: Main/urls.py
Comment (vanvo1605) [reply Main/urls.py:1]: gfgf

**Pull request merged**